### DISCUSSION TITLE
Your GitHub Personal Access Token has been revoked

### DISCUSSION THREAD

**Discussion by calumknight** [community/community#80976] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am running code on a cloud service. When I push my code to GitHub, I cannot use my account password and must instead use a Personal Access Token. When creating my Personal Access Token I set it to never expire as I do not want to have to create a new token each time I push code to GitHub. However, shortly after pushing my code using the token I have generated, I receive an email as follows:
"A recent scan found a valid GitHub Personal Access Token...linked to your GitHub account in the content of a commit. We have revoked it to protect your data from unauthorized access, and as a consequence, any app using this token won't be able to authenticate to GitHub."
How can I set up a Personal Access Token or other means of pushing my code to GitHub that does not require me to repeatedly return to GitHub's security settings to fix this problem?

**Comment by zspo** [comment]
Have you referred to this official documentation before?
https://docs.github.com/en/authentication/connecting-to-github-with-ssh

“Adding a new SSH key to your GitHub account” is important

**Reply by calumknight** [reply]
Hi, thanks for the information. I've tried to follow these instructions but unfortunately it's way over my head. Any chance there's a simplified 'how-to' guide for setting this kind of thing up?
